cli/config.go

```diff
@@ -74,6 +76,8 @@ func registerFlags(c *Config, f *flag.FlagSet) {
 	f.StringVar(&c.CAKeyFile, "ca-key", "", "CA private key")
 	f.StringVar(&c.TLSCertFile, "tls-cert", "", "Other endpoint CA to set up TLS protocol")
 	f.StringVar(&c.TLSKeyFile, "tls-key", "", "Other endpoint CA private key")
+	f.StringVar(&c.MutualTLSCAFile, "mutual-tls-ca", "", "Mutual TLS - require clients be signed by this CA ")
+	f.StringVar(&c.MutualTLSCN, "mutual-tls-cn", "", "Mutual TLS - require clients cert have the given CN")
```

Comment:
You need a good justification of this CN-checking functionality. Generally, CNs on client certs are meaningless and not controlled (and thus worthless for authentication).